codacy/values-production.yaml

```diff
@@ -240,6 +240,10 @@ crow:
     requests:
       cpu: 0.5
       memory: 1Gi
+  config:
```

Comment:
this is already the default because you put it in the `values.yaml` dosent need to be here